app/src/main/java/com/example/parkingsystemkotlin/mvp/presenter/ReservationParkingPresenter.kt

```diff
@@ -67,15 +67,13 @@ class ReservationParkingPresenter(
         var dateToString: String = Constant.EMPTY_STRING
         try {
             dateToString = simpleDateFormat.format(date)
-        } catch (e1: Exception) {
-            e1.printStackTrace()
+        } catch (exceptionCalendar: Exception) {
+            exceptionCalendar.printStackTrace()
         }
         return dateToString
     }
 
-    private fun getIntegerUserData(userData: String): Int {
-        return if (userData.isEmpty()) {
-            Constant.NOT_VALID
-        } else userData.toInt()
-    }
+    private fun getIntegerUserData(userData: String): Int = if (userData.isEmpty()) {
+        Constant.NOT_VALID
+    } else userData.toInt()
```

Comment:
can this fit on a single line?

Reply:
Yes, but when l `ctrl alt hago` change that, how can i change this?

Reply:
Done